### PR TITLE
Gibber will now give 3 pieces of meat at tier 1

### PR DESCRIPTION
**Pull request by jughu**
This been bugged for YEARS the gibber always gave 3 pieces of meat and then +2 on every tier ontop of that.
If you put a body in the gibber like a WHOLE human body and its gives 1 piece of meat its just said this fixes the issue thats been here for years

## Changelog
:cl: Improvedname
fix: Gibbers now produce 3 pieces of meat at tier 1 again
/:cl:

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
how is it a bug if you start with 0 and you know the ratings of base parts total to 1 your expected results are 1???

The previous PR fixed a bug where it would never reset so the first body would give you 1, second 2, third 3, etc. and if you upgraded the parts the matter only got worse.

File: code/modules/food_and_drinks/kitchen_machinery/gibber.dm [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 /obj/machinery/gibber/RefreshParts()
 	gibtime = 40
-	meat_produced = 0
+	meat_produced = 2
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/food_and_drinks/kitchen_machinery/gibber.dm:25]:
```suggestion
	meat_produced = initial(meat_produced)
```

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
in addition to changing L15

**Comment by ExcessiveUseOfCobblestone** [comment]
you need to change line 15 to `= 2` else itll always revert to 0 when you want 2(+1)

**Comment by Krysonism** [comment]
Meat is too easy to get already in my opinion.

Having to source meat from actual mobs or animal breeding would be more engaging gameplay.

**Comment by jughu** [comment]
This is litteraly the standard meat that always came from the gibber 6 years ago when it was made upgradeable

**Comment by jughu** [comment]
You litteraly butcher a monkey for the same amount of meat @Krysonism 
Gibber always gave 3 meat before stock parts became upgradable which reduced it into 1 then you had t2 which gave you 2 and at t4 you got 4 max.
With this fix everything gives 3 meat again and at t4 you get 7 meat at max

**Comment by ExcessiveUseOfCobblestone** [comment]
Food balance...

**Comment by ExcessiveUseOfCobblestone** [comment]
gave you CI because of the initial( change, as it minimizes the numbers people will have to change in the future.

**Pull request merged**